src/main/java/backend/resource/TurboLabel.java

```diff
@@ -4,40 +4,47 @@
 import backend.resource.serialization.SerializableLabel;
 import javafx.scene.Node;
 import javafx.scene.control.Tooltip;
+
 import org.eclipse.egit.github.core.Label;
+import util.Utility;
 
+import java.util.ArrayList;
+import java.util.List;
 import java.util.Optional;
 import java.util.regex.Matcher;
 import java.util.regex.Pattern;
+import java.util.stream.Collectors;
 
-@SuppressWarnings("unused")
 public class TurboLabel implements Comparable<TurboLabel> {
 
+    public static enum Grouping { EXCLUSIVE, NON_EXCLUSIVE, NONE }
+
+    public static final String DEFAULT_COLOR = "ffffff";
     public static final String EXCLUSIVE_DELIMITER = ".";
     public static final String NONEXCLUSIVE_DELIMITER = "-";
 
-    private void ______SERIALIZED_FIELDS______() {
-    }
-
     private final String actualName;
+    private final String simpleName;
+    private final String groupName;
```

Comment:
We can use these three terms to identify various parts of a label name: `fullName`, `shortName`, `groupName`. What do you think?